app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -63,7 +63,9 @@ protected void onCreate(Bundle savedInstanceState) {
         getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                 WindowManager.LayoutParams.FLAG_FULLSCREEN);
         setContentView(R.layout.activity_circuit);
-        wireController = new WireController((WireView) findViewById(R.id.draw_view));
+
+        int cellSize = getResources().getInteger(R.integer.cell_size);
+        wireController = new WireController((WireView) findViewById(R.id.draw_view), cellSize, cellSize);
```

Comment:
Wat is het nu van de verschillende cellWidth en cellHeigth variabelen als je hier twee keer dezelfde variabele als parameter meegeeft?